core/chains/solana/toml.go

```diff
@@ -0,0 +1,24 @@
+package solana
+
+import (
+	"github.com/smartcontractkit/chainlink/core/config/toml"
+	"github.com/smartcontractkit/chainlink/core/store/models"
+)
+
+type TOMLChain struct {
```

Comment:
I was planning to simplify one way or another after everything shakes out. `Config` should be available since we used `ChainCfg` before. One question still open is whether there are config values for a chain type broadly, rather than only specific chains of that type - in which case we'd want separate `Config` and `ChainConfig`, for example.

Reply:
I cleaned these up. I landed on `config` and `config/v2` packages, which may be overkill, but avoids naming clashing during the transition, and entangling too many imports. Plus for Terra and Solana, these can easily be moved to their respective repos once things have settled.